### ISSUE TITLE
salt/jinja2 Error with when installing in addon mode in SIFT

### ISSUE THREAD

**Issue by DanFelton** [REMnux/remnux-cli#72]
Hi,

I'm trying to install remnux in addon mode alongside SIFT on Ubuntu 20.04.

When running 'sudo remnux install --mode=addon' I get the following error:
![image](https://user-images.githubusercontent.com/1556263/164977794-ca2b3894-6f26-4543-807a-1b5765cedce6.png)

The contents of /var/cache/remnux/cli/v2022.16.1/saltstack.log are:
![image](https://user-images.githubusercontent.com/1556263/164977821-643a8eb2-43f4-4fe8-9f1b-993bb2bdce42.png)

I see other issues relating to jinja2 that were apparently fixed/closed but this is happening for me with the latest version of remnux-cli (1.3.4)

**Comment by DanFelton** [comment]
I saw in another thread that using v3.0.3 of jinja worked for someone else so I downgraded jinja2 (pip install -Iv jinja2==3.0.3) but that didn't make a difference and I got the same error when running the remnux install

**Comment by digitalsleuth** [comment]
Hi @DanFelton , did you use sudo for the pip install command? If not, give that a try.

**Comment by DanFelton** [comment]
Ok, I am not fully sure what's going on with pip, but trying to do sudo pip install -lv Jinja2==3.0.3 didn't work. Then I did pip uninstall Jinja2, which didn't fully uninstall it, but seem to have downgraded Jinja2 to v3.0.3 (according to the output of sudo pip list).

I tried doing a fresh pip install for Jinja 3.0.3 and that seems to have worked. I'm not very knowledgeable with pip and I'm still not 100% sure why it said I had 3.0.3 installed after uninstalling 3.1.1

Thanks for your help
